### PR TITLE
Delete unused settings

### LINKED ISSUE #47: Verify configuration settings are actually being used
With the vast amount of config settings that Argo uses, there are bound to be legacy settings that are not being used.

## Suspicious settings usage
 - [x] `STACKS_URL` & `STACKS_FILE_URL`

### LINKED ISSUE #48: Consolidate configuration for dev,test, prod environments
There seems to be opportunities to consolidate configuration files.

### PR DESCRIPTION
**Pull request by mejackreed**
Closes #47 
Closes #48 

This PR removes legacy duplicative `Argo::Config` pattern for setting configurations while also pruning away legacy configs that have snuck in that are not being used anymore.

### CODE REVIEW

**Comment by mejackreed** [comment]
Also happy to squash all of this down, but wanted to leave them apart for review. Let me know how you want it.

File: app/helpers/items_helper.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def valid_resource_types(type)
 
   def stacks_url_full_size(obj, file_name)
     druid = obj.pid
-    "#{Argo::Config.urls.stacks_file}/#{druid}/#{URI.encode(file_name)}"
+    "#{Settings.STACKS_URL_file}/#{druid}/#{URI.encode(file_name)}"
```
Comment (cbeer) [inline app/helpers/items_helper.rb]: Is this right, or find and replace gone wrong?

Comment (mejackreed) [inline app/helpers/items_helper.rb]: Ah.. great catch thanks.

Comment (mejackreed) [inline app/helpers/items_helper.rb]: :crying_cat_face: that our test suite didn't catch that.

**Comment by jmartin-sul** [comment]
:+1: 

i'd be slightly in favor of squashing, though also fine not squashing.  if you did squash, i think it'd be helpful to include each of the old commit messages as a line in the body of the squashed commit msg, with maybe the title of this PR as the subject line for the squashed commit msg.

**Comment by mejackreed** [comment]
I'm happy to squash down. Any further review before I do so?

**Comment by jmartin-sul** [comment]
nope, all good as far as i can tell, and a definite improvement

**Pull request merged**